frontend/src/utils/index.ts

```diff
@@ -3,4 +3,6 @@ import * as gps from './gps';
 import * as time from './time';
 import * as zones from './zones';
 
-export default { general, gps, time, zones };
+const utils = { general, gps, time, zones };
+
+export default utils;
```

Comment:
Kanskje litt utenfor, men hva er gjør egentlig denne filen?

Reply:
It doesn't really do anything - it just makes it possible to explicitly import all the utils functions to the same object, and then explicitly use i.e. `utils.gps.functionName` - Might be a bit overkill, but i say we try this setup and evenrually change the stuff later